templates/score.html

```diff
@@ -0,0 +1,26 @@
+<p>Results are here:</p>
+<!-- Parādām mūsu faila rindas -->
+<UL  id="scores">
+{% for value in lines %}
+<li>
+    {{ value }}
+</li>
+
+{% endfor %}
+</UL>
+<table>
+    <tbody id="tabula"></tbody>
+</table>
+ <script>
```

Comment:
Kāda bija ideja šīm te manipulācijām? Gribējās uzzīmēt html tabulu ar JS? Ja tā, tad man šķiet, ka tas šeit ir lieki.

Augstāk jau bija pareizais virziens, laikam nesanāca no rindiņas dabūt ārā kolonnas.
Cikls jau bija korekts, piemēram:
```html
    <table>
      <thead>
          <tr>
            <th>Vārds</th>
            <th>Bilance</th>
            <th>Iznākums</th>
          </tr>
      </thead>
      <tbody>
          {% for line in lines %}
          {% set items = line.split() %}
            <tr>
              <td>{{ items[0] }}</td>
              <td>{{ items[1] }}</td>
              <td>{{ items[2] }}</td>
            </tr>
          {% endfor %}
      </tbody>
    </table>
```

Ciklā izejam cauri katrai failā ierakstītajai rindai.
Tā kā mums failā glabājas rindas, tad gribās piekļūt katrai rindas kolonnai, ko var izdarīt ar:
```python
 line_columns = line.split()
```
kur funckija split() simbolu virkni sadalīs masīvā pēc " " tukšuma simbola.
Piemēram, line = "A B C", tad line_columns = ["A", "B", "C"]. Un pēc tam tad mēs varam piekļūt klāt šīm te kolonnām ar attiecīgo indeksu.

Reply:
Jā, gribējās uzzīmēt tabulu ar JS un pēc katra pieprasījuma serverim, tiktu atgriezti rezultāti tabulā.

Reply:
Labi, bet tad sākumā jābūt tukšai tabulai. Tad ar fetch var veikt pieprasījumu. Pie reizes tad vajadzētu realizēt to, ka datus prasa, piemēram, pa 10 ierakstiem. Servera pusē tad tas arī jāapstrādā. Ja vēl gribās meklēšanu, kārtošanu, tad tas arī.